app/models/creator.rb

```diff
@@ -1,10 +1,28 @@
 # frozen_string_literal: true
 
 class Creator < ApplicationRecord
-  # TODO validations? Indexes?
   has_many :work_version_creations,
            dependent: :restrict_with_exception
 
   has_many :work_versions,
-           through: :work_version_creations
+           through: :work_version_creations,
+           inverse_of: :creators
+
+  validates :email,
+            presence: true
```

Comment:
@rschenk I'm wondering if surname should required too. One of the things Scholarsphere does is parse a display name for given and surnames: https://github.com/psu-stewardship/scholarsphere/blob/develop/app/forms/creator_form.rb#L23 That way there's at least something in the field. This will affect indexing, because we want to facet on the creator name, and not the alias. Just having an email would make that weird.

Reply:
🏈